docs/SECURITY.md

```diff
@@ -0,0 +1,12 @@
+#### Reporting a Vulnerability
+
+If you find a vulnerability in the code or the behavior of the application that is not very critical then create an [issue on the repository](https://github.com/fossunited/fossunited/issues). with all the possible details. An issue on the GitHub repo should be created only if the vulnerability or bug is not very critical.
+
+In case, the Bug or Vulnerability is very critical and deals directly with user data and information which can certainly cause a 0-day attack, then please send us an email at security@fossunited.org and the team shall get back to you within 12-24 hours. 
```

Comment:
```
In case, the (suspected) bug or vulnerability is very critical and may leak data, then please send us an email at security@fossunited.org and the team shall get back to you within 12-24 hours. Furthermore, a resolution timeline will be shared.
```

Is much better. The `0-day` line barely makes sense.